src/qt/sendcoinsdialog.cpp

```diff
@@ -306,15 +306,12 @@ bool SendCoinsDialog::PrepareSendText(QString& question_string, QString& informa
         formatted.append(recipientElement);
     }
 
-    if (model->wallet().privateKeysDisabled()) {
-        question_string.append(tr("Do you want to draft this transaction?"));
-    } else {
-        question_string.append(tr("Are you sure you want to send?"));
-    }
-
     question_string.append("<br /><span style='font-size:10pt;'>");
+    question_string.append(tr("Do you want to create this transaction?"));
     if (model->wallet().privateKeysDisabled()) {
         question_string.append(tr("Please, review your transaction proposal. This will produce a Partially Signed Bitcoin Transaction (PSBT) which you can save or copy and then sign with e.g. an offline %1 wallet, or a PSBT-compatible hardware wallet.").arg(PACKAGE_NAME));
+    } else if (model->getOptionsModel()->getEnablePSBTControls()) {
+        question_string.append(tr("Please, review your transaction. You can create and send this or transaction. Or you can create a Partially Signed Bitcoin Transaction (PSBT) which you can save or copy and then sign with e.g. an offline %1 wallet, or a PSBT-compatible hardware wallet. Or you can ").arg(PACKAGE_NAME));
```

Comment:
PSBTs aren't Bitcoin Core specific, though, so `PACKAGE_NAME` seems to be the wrong substitution to use here?

Reply:
This is just the same as the string above.